src/interactive/repl.ts

```diff
@@ -336,6 +336,17 @@ async function evaluateBlockOrSelection(shouldMove: boolean = false) {
             editor.revealRange(new vscode.Range(nextBlock, nextBlock))
         }
 
+
+        const tempDecoration = vscode.window.createTextEditorDecorationType({
+            backgroundColor: new vscode.ThemeColor("editor.hoverHighlightBackground"),
+            isWholeLine: true
+        })
+        editor.setDecorations(tempDecoration, [range])
+
+        setTimeout(() => {
+            editor.setDecorations(tempDecoration, [])
+        }, 200)
+
```

Comment:
before we have a progress metre, how about just clearing decoration after `await evaluate(editor, range, text, module)` rather than flashing ?
It may look kinda verbose if code runs long time, but it also indicates code is being executed.

Please discard this idea if you don't like this, I even not sure if I like that.

Reply:
nvm, we've not established the way to interruption mechanism yet, so simply there won't be no way to clear the temporary decoration if code never terminates.
Introducing interruption should be done in separate PR. Let's keep this as flashing for now.

Reply:
![Peek 2020-06-03 13-14](https://user-images.githubusercontent.com/6735977/83630461-21df8780-a59c-11ea-89db-691e678a6f2b.gif)

Not sure if I like it or not...

Reply:
yeah, me neither, let's ignore this lol